neptune/client.py

```diff
@@ -214,24 +217,24 @@ def create_experiment(self, project, name, description, params, properties, tags
             else:
                 raise
 
+    @with_api_exceptions_handler
+    def get_experiment(self, experiment_id):
+        return self.backend_swagger_client.api.getExperiment(experimentId=experiment_id).response().result
+
     @with_api_exceptions_handler
     def update_experiment(self, experiment, properties):
```

Comment:
1. `experiment` argument is used only to get its `internal_id` and as return value. Let's just pass `experiment_id` instead.
2. How do we update tags, name, description, etc.?
3. Shouldn't this method be called `update_properties`?

Reply:
1. It's not. `experiment.project_full_id` is used in 404 (NotFound Error) handling.
2. Tags: there is a separate method below, `update_tags`. Name, description: currently, we don't.
3. It's a wrapper for `updateExperiment` api method so I think that current method name looks better than `updateProperties`.

Reply:
3. But it updates only properties.